lib/rubocop/cop/lint/mixed_case_range.rb

```diff
@@ -79,7 +79,7 @@ def range_for(char)
         end
 
         def range_pairs(expr)
-          RuboCop::Cop::Utils::RegexpRanges.new(expr).pairs
+          expr.expressions.select { |e| e.type == :set }.map { |e| [e.expressions[0], e.expressions[1]] }
```

Comment:
To also avoid the `Layout/LineLength` cop's offense, the code could be written as follows.

```suggestion
          expr.expressions.filter_map { |e| [e.expressions[0], e.expressions[1]] if e.type == :set }
```

Reply:
That's nice, thanks